bahmni-docker/.env

```diff
@@ -130,3 +130,14 @@ PACS_SIMULATOR_TIMEOUT=20000
 PACS_SERVER_TYPE=dcm4chee
 PACS_SERVER_URL=DCM4CHEE@dcm4chee:11112
 UPDATE_PACS_INTEGRATION_DB=true
+
+#Crater-Atomfeed Environment variables
+CRATER_USERNAME=admin@craterapp.com
+CRATER_PASSWORD=crater@123
+CRATER_ATOMFEED_DB_USERNAME=beehyv
+CRATER_ATOMFEED_DB_PASSWORD=beehyv123
+CRATER_ATOMFEED_DB_ROOT_PASSWORD=beehyv123
+CRATER_ATOMFEED_DB_HOST=crater-atomfeed-db
+CRATER_ATOMFEED_DB_PORT=3306
+CRATER_ATOMFEED_DB_NAME=crater_atomfeed
+CRATER_URL=http://localhost:81
```

Comment:
@stavtnameh With this configuration is the atomfeed sync working with local openmrs and crater ? I feel that the URL would need a change. `localhost` would resolve within the container. Please test it.